src/main/java/core/jdbc/JdbcTemplate.java

```diff
@@ -0,0 +1,92 @@
+package core.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class JdbcTemplate {
+
+    public int update(PreparedStatementCreator creator, PreparedStatementSetter setter) {
+        try {
+            try (Connection con = ConnectionManager.getConnection()) {
+                try (PreparedStatement statement = creator.createPreparedStatement(con)) {
```

Comment:
불필요한 삼중 try 문이 들어가있네요.
```suggestion
                PreparedStatement statement = creator.createPreparedStatement(con);
```